### ISSUE TITLE
NODE_HOME environment variable is ignored by `zwe install`

### ISSUE THREAD

**Issue by plavjanik** [zowe/zowe-install-packaging#2904]
**Describe the bug**
I am doing a fresh install of Zowe on a system without having Node.js in the PATH.

**To Reproduce**

1. Follow the installation instructions including adding `zwe` to PATH and defining ZWE_CLI_PARAMETER_CONFIG 
2. Run `zwe install`
3. It fails with `Error ZWEL0121E: Cannot find node. Please define NODE_HOME environment variable.`
4. Try to define NODE_HOME environment variable in such a way that $NODE_HOME/bin/node works. Unable to find documentation about what needs to be in NODE_HOME
5. It fails too
6. Try to define NODE_HOME so $NODE_HOME/node works
7. It fails too
8. Try adding $NODE_HOME/bin to the PATH
9. Node.js located

**Expected behavior**

The error message should explain what is a valid value for NODE_HOME and it should be used.

**Logs**

```
$ zwe install
===============================================================================
>> INSTALL ZOWE MVS DATA SETS

Error ZWEL0121E: Cannot find node. Please define NODE_HOME environment variable.
$ export NODE_HOME=/usr/lpp/IBM/cnj/v14r0/IBM/node-v14.18.3-os390-s390x-202202141428
$ echo $NODE_HOME
/usr/lpp/IBM/cnj/v14r0/IBM/node-v14.18.3-os390-s390x-202202141428
$ zwe install
===============================================================================
>> INSTALL ZOWE MVS DATA SETS

Error ZWEL0121E: Cannot find node. Please define NODE_HOME environment variable.
$ $NODE_HOME/bin/node
Welcome to Node.js v14.18.3.
Type ".help" for more information.
>
(To exit, press Ctrl+C again or Ctrl+D or type .exit)
>
$ export PATH=$NODE_HOME/bin:$PATH
$ node
Welcome to Node.js v14.18.3.
Type ".help" for more information.
>
(To exit, press Ctrl+C again or Ctrl+D or type .exit)
>
$ zwe install
===============================================================================
>> INSTALL ZOWE MVS DATA SETS

Create MVS data sets if they do not exist
```

**Comment by jackjia-ibm** [comment]
Is this a duplicate of https://github.com/zowe/zowe-install-packaging/issues/2839?

**Comment by plavjanik** [comment]
> Is this a duplicate of #2839?

It looks like the same issue.

**Comment by jackjia-ibm** [comment]
The fix should be shipped with v2.1. Will close the issue. Please reopen if the issue still exists.
